utils/check_copies.py

```diff
@@ -186,6 +187,7 @@ def find_code_in_transformers(object_name: str) -> str:
 
 
 _re_copy_warning = re.compile(r"^(\s*)#\s*Copied from\s+transformers\.(\S+\.\S+)\s*($|\S.*$)")
+_re_copy_warning_for_test_file = re.compile(r"^(\s*)#\s*Copied from\s+models\.(\S+\.\S+)\s*($|\S.*$)")
```

Comment:
```suggestion
_re_copy_warning_for_test_file = re.compile(r"^(\s*)#\s*Copied from\s+tests\.(\S+\.\S+)\s*($|\S.*$)")
```
should look for tests/models? or is it because of the path?

Reply:
it depends on the base. For test files, the base path is `MODEL_TEST_PATH = "tests/models"`, and in this case, the reg. exp. has to be `models`. (the `models` in `tests`).

I can of course change the base path to `MODEL_TEST_PATH = "tests"`, but it will try to check  all the files in `tests` which means more files (and not really our goal here and wasting time)

Reply:
ok no worries